doc/userguide/upgrade/8.0-dns-logging-changes.rst

```diff
@@ -0,0 +1,260 @@
+:orphan: Document not referenced in a toctree, so add this.
```

Comment:
What does this `orphan` mean ?

Reply:
Sphinx will complain if a document exists but is not reference in a toctree (table of contents). This document is only linked from its upgrade note, so we suppress the error with this.